tests/tests_unit/graph/test_transformers/test_asset_depth_transformer.py

```diff
@@ -24,6 +24,30 @@ def test_asset_depth_transformer():
     )
 
 
+def test_asset_depth_transformer_with_typing():
+    store = NeatGraphStore.from_memory_store()
+    extractor = extractors.AssetsExtractor.from_file(CLASSIC_CDF_EXTRACTOR_DATA / "assets.yaml")
+    store.write(extractor)
+
+    transformer = transformers.AddAssetDepth(
+        depth_typing={
+            1: "RootCimNode",
+            2: "GeographicalRegion",
+            3: "SubGeographicalRegion",
+            4: "Substation",
+        }
+    )
+
+    store.transform(transformer)
+
+    summary = dict(zip(store.summary.iloc[:, 0], store.summary.iloc[:, 1], strict=False))
+
```

Comment:
I also added `store.queries.list_types()` that you can use.

Reply:
cool! I will move summary query there